Makefile

```diff
@@ -64,3 +64,13 @@ test: build/xdebug-filter.php
 docs:
 	cd docs && sphinx-build -W -b html -d _build/doctrees . _build/html
 .PHONY: docs
+
+yarn-install:
+	@docker run --rm -it -v ${PWD}:/usr/src/app -w /usr/src/app node:11-alpine yarn install
```

Comment:
You could have a `YARN` make variable to store `docker run --rm -it -v ${PWD}:/usr/src/app -w /usr/src/app node:11-alpine yarn`

Reply:
Nice idea!

Reply:
Plus that way people who want to avoid docker will be able to.